shared/attributes.asciidoc

```diff
@@ -358,24 +358,26 @@ Common words and phrases
 :nlp:                        natural language processing
 :nlp-cap:                    Natural language processing
 
-:apm-go-agent:               Elastic APM Go Agent
-:apm-go-agents:              Elastic APM Go Agents
-:apm-ios-agent:              Elastic APM iOS Agent
-:apm-ios-agents:             Elastic APM iOS Agents
-:apm-java-agent:             Elastic APM Java Agent
-:apm-java-agents:            Elastic APM Java Agents
-:apm-dotnet-agent:           Elastic APM .NET Agent
-:apm-dotnet-agents:          Elastic APM .NET Agents
-:apm-node-agent:             Elastic APM Node.js Agent
-:apm-node-agents:            Elastic APM Node.js Agents
-:apm-php-agent:              Elastic APM PHP Agent
-:apm-php-agents:             Elastic APM PHP Agents
-:apm-py-agent:               Elastic APM Python Agent
-:apm-py-agents:              Elastic APM Python Agents
-:apm-ruby-agent:             Elastic APM Ruby Agent
-:apm-ruby-agents:            Elastic APM Ruby Agents
-:apm-rum-agent:              Elastic APM Real User Monitoring (RUM) JavaScript Agent
-:apm-rum-agents:             Elastic APM RUM JavaScript Agents
+:apm-server:                 APM Server
```

Comment:
I forgot that we set and unset an `apm-server` attribute in the Beats docs to manage included content. This duplicates that attributes which causes a cascade of `includes` to fail.

```suggestion
```